data/migrations/v1.0.0-point.sql

```diff
@@ -10,3 +10,39 @@ UPDATE
      tags -> 'power' IN ('plant', 'substation') OR
      tags -> 'natural' IN ('wood', 'forest'))
     AND COALESCE(mz_poi_min_zoom, 999) <> COALESCE(mz_calculate_min_zoom_pois(planet_osm_point.*), 999);
+
+CREATE INDEX new_planet_osm_point_min_zoom_pois_index ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom IS NOT NULL;
+CREATE INDEX new_planet_osm_point_min_zoom_pois_6_index ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 6;
+CREATE INDEX new_planet_osm_point_min_zoom_pois_9_index ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 9;
+CREATE INDEX new_planet_osm_point_min_zoom_pois_12_index ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 12;
+CREATE INDEX new_planet_osm_point_min_zoom_pois_15_index ON planet_osm_point USING gist(way) WHERE mz_poi_min_zoom <= 15;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_index RENAME TO planet_osm_point_min_zoom_pois_index;
+COMMIT;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_6_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_6_index RENAME TO planet_osm_point_min_zoom_pois_6_index;
+COMMIT;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_9_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_9_index RENAME TO planet_osm_point_min_zoom_pois_9_index;
+COMMIT;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_12_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_12_index RENAME TO planet_osm_point_min_zoom_pois_12_index;
+COMMIT;
+
+BEGIN;
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_pois_15_index;
+ALTER INDEX new_planet_osm_point_min_zoom_pois_15_index RENAME TO planet_osm_point_min_zoom_pois_15_index;
+COMMIT;
+
+DROP INDEX IF EXISTS planet_osm_point_min_zoom_way_index;
```

Comment:
I think the existing index is called `planet_osm_point_min_zoom_wa9_index` for the `IS NOT NULL` constraint.